mkt/lookup/views.py

```diff
@@ -387,7 +387,7 @@ def _expand_query(q, fields):
 def user_search(request):
     results = []
     q = request.GET.get('q', u'').lower().strip()
-    search_fields = ('username', 'display_name', 'email')
+    search_fields = ('fxa_uid', 'display_name', 'email')
```

Comment:
Above you remove `display_name` from the template, should we remove/change it here as well? Is it used and worthy of querying?